crates/net/network/src/transactions.rs

```diff
@@ -857,7 +856,7 @@ where
                     // known that this transaction is bad. (e.g. consensus
                     // rules)
                     if err.is_bad_transaction() && !this.network.is_syncing() {
-                        trace!(target: "net::tx", ?err, "Bad transaction import");
+                        trace!(target: "net::tx", ?err, "bad pool transaction import");
```

Comment:
can we output tx hash here as well? it would help us inspect them in case we have false positives

Reply:
this is already part of the error:

https://github.com/paradigmxyz/reth/blob/d08480a28501fdc58f3a3c3a47e607d949ee1ab4/crates/transaction-pool/src/error.rs#L29-L29